src/System.Security.Cryptography.Encoding/ref/System.Security.Cryptography.Encoding.cs

```diff
@@ -66,4 +66,29 @@ public enum OidGroup
         SignatureAlgorithm = 4,
         Template = 9,
     }
+
```

Comment:
Do we need to bump a version number on the contract?  Or because these contracts aren't yet stable, we don't?